packages/react-ux-capture-example/src/Bar.jsx

```diff
@@ -17,11 +27,30 @@ class Bar extends Component {
 				primaryContentDisplayed={primaryContentDisplayed}
 			>
 				<div className="chunk">
-					<h1 className="text--pageTitle">Bar Lazy Title</h1>
-					<Inline mark="ux-text-bar-title" />
+					<h1
+						className="text--pageTitle"
+						style={getBoxStyle('ux-destination-verified')}
+					>
+						Lazy Bar Title
+					</h1>
+					<UXCaptureInlineMark mark="ux-text-bar-title" />
+				</div>
+				<div className="chunk">
+					This view only defines two zones and cuptures corresponding
```

Comment:
A continuation of my previous comment about file/page naming would be that I'd also lean toward real content as opposed to content that describes functionality and timings. 

You can use code comments to explain things like `This view only defines two zones and captures ...`, or `This is lazy loaded`, etc.

Reply:
I would disagree here, self-documenting examples are useful.

Page itself describing what it is for is useful.

Besides, "real" content is lorem ipsum anyway ;)

Reply:
The point is that we are not building real app that makes data requests and stuff and therefore legitimately takes longer to load things.

I can spell it out in comments, but these Lazy-loaded components only emulate things.

Same goes for content, it only pretends to be real to show variety of cases rather than having meaning on their own.